pages/linux/jpegtran.md

```diff
@@ -0,0 +1,28 @@
+# jpegtran
+
+> Perform lossless transformation of JPEG files.
+> More information: <https://linux.die.net/man/1/jpegtran>.
+
+- Mirror image horizontally or vertically:
+
+`jpegtran -flip {{horizontal|vertical}} {{/path/to/image.jpg}}`
+
+- Rotate image 90, 180 or 270 degrees clockwise:
+
+`jpegtran -rotate {{90|180|270}} {{/path/to/image.jpg}}`
+
+- Transpose the image across the upper left to lower right axis:
+
+`jpegtran -transpose {{/path/to/image.jpg}}`
+
+- Transverse the image across the upper right to lower left axis:
+
+`jpegtran -transverse {{/path/to/image.jpg}}`
+
+- Convert the image to grayscale:
+
+`jpegtran -grayscale {{/path/to/image.jpg}}`
+
+- Crop the image to a rectangular region of width `W` and height `H`, starting at point `X` and `Y`:
```

Comment:
We should say which is the origin (point 0,0) of the coordinate system. Maybe this is better:
```suggestion
- Crop the image to a rectangular region of width `W` and height `H`, starting at point `X` and `Y` from the upper left corner:
```